ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/PromptTemplate.java

```diff
@@ -5,10 +5,10 @@ public class PromptTemplate {
+        "Human:RESPONSE FORMAT INSTRUCTIONS\n----------------------------\nOutput a JSON markdown code snippet containing a valid JSON object in one of two formats:\n\n**Option 1:**\nUse this if you want the human to use a tool.\nMarkdown code snippet formatted in the following schema:\n\n```json\n{\n    \"thought\": string, // think about what to do next: if you know the final answer just return \"Now I know the final answer\", otherwise suggest which tool to use.\n    \"action\": string, // The action to take. Must be one of these tool names: [${parameters.tool_names}], do NOT use any other name for action except the tool names. If the required tool is not part of available tools then use Option 2 to generate final response with available information.\n    \"action_input\": string // The input to the action. May be a stringified object.\n}\n```\n\n**Option #2:**\nUse this if you want to respond directly and conversationally to the human. Markdown code snippet formatted in the following schema:\n\n```json\n{\n    \"thought\": \"Now I know the final answer\",\n    \"final_answer\": string, // summarize and return the final answer in a sentence with details, don't just return a number or a word. \n}\n```";
     public static final String PROMPT_TEMPLATE_SUFFIX =
         "Human:TOOLS\n------\nAssistant can ask the user to use tools to look up information that may be helpful in answering the users original question. The tools the human can use are:\n\n${parameters.tool_descriptions}\n\n${parameters.prompt.format_instruction}\n\n${parameters.chat_history}\n\n\nHuman:USER'S INPUT\n--------------------\nHere is the user's input (remember to respond with a markdown code snippet of a json blob with a single action, and NOTHING else):\n${parameters.question}\n\n${parameters.scratchpad}";
     public static final String PROMPT_TEMPLATE = "\n\nHuman:${parameters.prompt.prefix}\n\n${parameters.prompt.suffix}\n\nAssistant:";
     public static final String PROMPT_TEMPLATE_TOOL_RESPONSE =
-        "TOOL RESPONSE: \n---------------------\n${parameters.observation}\n\nUSER'S INPUT\n--------------------\n\nOkay, so what is the response to my last comment? If using information obtained from the tools you must mention it explicitly without mentioning the tool names - I have forgotten all TOOL RESPONSES! Remember to respond with a markdown code snippet of a json blob with a single action, and NOTHING else.";
+        "TOOL RESPONSE: \n---------------------\n${parameters.observation}\n\nUSER'S INPUT\n--------------------\n\nOkay, so what is the response to my last comment? If using information obtained from the tools you must mention it explicitly without mentioning the tool names. Remember to respond with a markdown code snippet of a json blob with a single action, and NOTHING else.";
```

Comment:
Not sure if you want to remove this prompt, seems it's added on purpose.

Reply:
@jngz-es do you know why this prompt was added? I ran into few issues with this prompt where LLM said it no longer has access to tool output and it was not able to generate a valid response.

Reply:
Arjun, do you have examples to trigger the no access to tool problem? I was able to reproduce it consistently before, but it has been fixed by your fix to PROMPT_FORMAT_INSTRUCTION. I reverted your change to PROMPT_TEMPLATE_TOOL_RESPONSE and it continues to work fine.

Reply:
I was not able to consistently reproduce the problem, even before applying the fix I was only transiently running into the bug.